Créer un tableau de bord interne

ReX-AV-Gab opened the pull request:
Je vous propose de valider le principe et la structuration d’un tableau de bord interne avec possibilité de l’amender. Une fois que validation faite, nous pourrons faire vivre le tableau de bord (après présentation à l’équipe).
https://roamresearch.com/#/app/virtual-assembly/page/s2_xhfB53

simonLouvet committed:
PUT implemntation

simonLouvet committed:
merge

simonLouvet committed:
Merge branch 'master' into 151_PUT_LDP

simonLouvet committed:
finalise test and integration of PU

simonLouvet committed:
Automatic prettier

simonLouvet committed:
finalise test and integration of PUT

simonLouvet committed:
Merge branch '151_PUT_LDP' of github.com:assemblee-virtuelle/semapps into 151_PUT_LDP

simonLouvet committed:
Merge pull request #223 from assemblee-virtuelle/151_PUT_LDP

PUT implementation

simonLouvet committed:
midlle level ldp operation migration to ldp service + clen up logic + spimplify patch

simonLouvet committed:
Automatic prettier

simonLouvet committed:
remove log

simonLouvet committed:
remove log

simonLouvet committed:
review integration

simonLouvet committed:
Automatic prettier

simonLouvet committed:
review integration

simonLouvet committed:
Merge pull request #228 from assemblee-virtuelle/156_clean_triplestore_ldp_separation

middle level SPARQL computing migration to ldp service + claen up logic + spimplify patch

srosset81 committed:
Replace expand param with queryDepth

srosset81 committed:
Automatic prettier

srosset81 committed:
Format query

srosset81 committed:
Change expand to queryDepth for actor service too

srosset81 committed:
Add query param to ldp.container.get action

srosset81 committed:
Add LDP container service tests

srosset81 committed:
Fix LDP API tests

srosset81 committed:
Automatic prettier

srosset81 committed:
Fix AP/WebID tests

srosset81 committed:
Merge remote-tracking branch 'origin/ldp-container-tests' into ldp-container-tests

srosset81 committed:
Add missing await

srosset81 committed:
Automatic prettier

srosset81 committed:
Merge pull request #231 from assemblee-virtuelle/handle-blank-nodes-query

Replace expand param with queryDepth

srosset81 committed:
Merge pull request #232 from assemblee-virtuelle/ldp-container-get-query-params

Add query param to ldp.container.get action

srosset81 committed:
Merge pull request #236 from assemblee-virtuelle/ldp-container-tests

Add LDP container service tests

srosset81 committed:
Various improvements

srosset81 committed:
Prettier

bouviermullerp commented:
@ReX-AV-Gab je ne suis pas sur de bien comprendre la finalité d'un tableau de bord interne.
Peut-on éventullement utiliser les issues et le kanban, plutôt qu'un outil supplémentaire (en fonction de la finalité) ?

ReX-AV-Gab commented:
La finalité est de pouvoir accéder aux informations internes sans avoir à chercher pendant un trop long moment. Je pense que cela peut être très intéressant pour un nouvel arrivant dans l'équipe.
Si l'utilité n'est pas avérée, je pense qu'il serai alors préférable éviter d'utiliser un "outil" supplémentaire.

srosset81 committed:
Handle Unfollow AP activities

srosset81 committed:
Rename tests

srosset81 committed:
Automatic prettier

srosset81 commented:
@ReX-AV-Gab Les PR permettent de modifier des fichiers (code ou docusaurus), en proposant de merger deux branches. Ce n'est pas adapté s'il n'y a pas de code (d'ailleurs tu as été obligé de faire une comparaison entre deux branches qui n'avaient pas de sens)

Il faut faire une issue à la limite.

Désolé si c'est moi qui n'est pas été assez clair.

srosset81 committed:
Merge pull request #238 from assemblee-virtuelle/activitypub-unfollow

Handle Undo>Follow AP activities

simonLouvet committed:
change minor version

simonLouvet committed:
change minor version

simonLouvet committed:
Publish

 - @semapps/activitypub@0.1.4
 - @semapps/connector@0.1.4
 - @semapps/ldp@0.1.4
 - @semapps/mime-types@0.1.4
 - @semapps/sparql-endpoint@0.1.4
 - @semapps/triplestore@0.1.4
 - @semapps/webhooks@0.1.4
 - @semapps/webid@0.1.4

simonLouvet committed:
force npm publisf from git

simonLouvet committed:
force fuseki admmin version upgrade

simonLouvet committed:
change boilerplates runner dependencies version

srosset81 commented:
@ReX-AV-Gab Je ferme cette PR qui crée du bruit puisque chaque fois que quelque chose est ajouté sur master, ça crée une notifications.
Je t'invite à créer une issue pour ça si tu penses que c'est pertinent.
